docs/book/src/plugins/grafana-v1-alpha.md

```diff
@@ -0,0 +1,96 @@
+# Grafana Plugin
+
+The Grafana plugin provides the Grafana manifest that you can use to visualize your operator status easily.
+
+<aside class="note">
+<h1>Examples</h1>
+
+Samples are provided under the [testdata][testdata] directory of the Kubebuilder project.
+You can check samples using this plugin by looking at the `project-v3-with-grafana` projects under the [testdata][testdata] directory on the root directory of the Kubebuilder project.
+
+</aside>
+
+## When to use it ?
+
+- If you want to observe your operator runtime status based on the [controller metrics][controller-metrics] collected by Prometheus
+
+## How to use it ?
+
+### Prerequisites:
+
+1. Prometheus monitoring stacks running in your K8s cluster (e.g: [Prometheus Operator][prom-operator], [kube-prometheus][kube-prometheus])
+2. A servicemonitor that exposes your operator metrics to Prometheus
+   - If you're using the kustomize plugin in the meantime, you should have the manifest under `config/prometheus`
+3. Users must have write access to their grafana services
+
+### Sample of servicemonitor
```

Comment:
The service monitor is a pre-requirement for we are able to export the metrics to Prometheus Operator right?
So, if the person installed the Prometheus Operator as suggested in the doc will have the ServiceMonitor scaffold in the config and be able to use it. So, I think we should not add this example here.